launch/map4_localization_launch/README.md

```diff
@@ -0,0 +1,44 @@
+# map4_localization_launch
+
+This localization launch package is a sample package to introduce eagleye into autoware.
+
+Eagleye provides a cost-effective alternative to LiDAR and point cloud-based localization by using low-cost GNSS and IMU sensors to provide vehicle position and orientation.
+
+Autoware users will be able to choose between their existing LiDAR and point cloud-based localization stacks or GNSS/IMU-based eagleye localizer, depending on their specific needs and operating environment.
+
+There are two ways to utilize eagleye results with the autoware localization stack.
```

Comment:
It is kind of confusing here where it mentions "two" while in the next table there are three rows. How about this?
```suggestion
In addition to the LiDAR-based solution provided by `tier4_localization_launch` (shown in the first row of the following table), there are two ways to utilize eagleye in `map4_localization_launch` with the autoware localization stack.
```

Reply:
readme.md has been corrected.

Reply:
@yukkysaito @mitsudome-r 
Based on the discussion in the following link, changes have been made to the launch structure. Could you please check if there are any issues and provide your opinion?
https://github.com/autowarefoundation/autoware.universe/pull/3063#issue-1620972228

Reply:
@yukkysaito @mitsudome-r 
How about this PR?

Reply:
It's ok :+1: 

@mitsudome-r How about you?

Reply:
@mitsudome-r I am just confirming, ok?

Reply:
@kminoda 
Hello. As per the comments in this PR, I have just received an approval from @yukkysaito @mitsudome-r .